sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AADAuthenticationProperties.java

```diff
@@ -106,19 +103,20 @@
      */
     private long jwkSetCacheRefreshTime = DEFAULT_JWK_SET_CACHE_REFRESH_TIME;
 
-    private String postLogoutRedirectUri;
-
     /**
-     * If Telemetry events should be published to Azure AD.
+     * The redirect uri after logout.
      */
-    private boolean allowTelemetry = true;
+    private String postLogoutRedirectUri;
 
     /**
      * If true activates the stateless auth filter AADAppRoleStatelessAuthenticationFilter. The
      * default is false which activates AADAuthenticationFilter.
      */
     private Boolean sessionStateless = false;
 
+    /**
+     * The OAuth2 authorization clients.
+     */
```

Comment:
Later we can have more hints added for this map https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html#configuration-metadata.manual-hints.value-hint

Reply:
Issue created: https://github.com/Azure/azure-sdk-for-java/issues/26600